includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -382,7 +358,7 @@ public function data_table_header() {
 							)
 						);
 						?>
-						<button type="submit" id="sensei-bulk-learner-actions-modal-toggle" class="button button-primary action" disabled><?php echo esc_html__( 'Select Courses', 'sensei-lms' ); ?></button>
+						<div class="student-bulk-action-button" style="margin-left: 4px"></div>
```

Comment:
I think we need to prepend the class name with "sensei"?

Reply:
Hey, yes I renamed it here: https://github.com/Automattic/sensei/pull/5004/commits/2bf5e8d01a04d0028e9fa17b5fdcfa2d992530b0